iOS/Flows/New Conversation/ConversationCoordinator.swift

```diff
@@ -36,6 +37,9 @@ class ConversationCoordinator: PresentableCoordinator<Void> {
     }
 
     override func toPresentable() -> DismissableVC {
+        self.conversationVC.onSelectedThread = { [unowned self] (channelID, messageID) in
+            self.presentThread(for: channelID, messageID: messageID)
+        }
```

Comment:
What if we have three different handlers? Might be cleaner to move to a protocol delegate? No need to change now, just some thoughts.